modules/core_project_factory/scripts/preconditions/preconditions.py

```diff
@@ -243,19 +243,17 @@ def validate(self, credentials):
             'serviceusage', 'v1',
             credentials=credentials
         )
-
         parent = "projects/" + self.project_id
-        request = service.services().list(
-            parent=parent,
-            filter='state:ENABLED'
-        )
+        enabled = []
+        for REQUIRED_API in self.REQUIRED_APIS:
```

Comment:
The iterator value should not be represented as a constant.

```suggestion
        for required_api in self.REQUIRED_APIS:
```